apps/test/unit-tests.js

```diff
@@ -26,5 +26,6 @@ describe('unit tests', function () {
   throwOnConsoleErrorsEverywhere();
   throwOnConsoleWarningsEverywhere();
   clearTimeoutsBetweenTests();
+  // enforceDocumentBodyCleanup({checkEveryTest: true});
```

Comment:
Included to point out where I've been calling my helper.  To actually use this you also need to import it at the top of the file.

Reply:
Unless you're sure this will only live a short time, an explanatory comment would be nice. How about: "For now, uncomment and import this to perform additional test cleanup validation. In the long term this should be permanently enabled or removed."